docs/requirements.md

```diff
@@ -2,88 +2,76 @@
 
 Running Codacy on a Kubernetes cluster requires the following:
 
--   A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#cluster-resource-requirements)
--   The [NGINX Ingress Controller](https://github.com/helm/charts/tree/master/stable/nginx-ingress) for Kubernetes
+-   A Kubernetes 1.14.\* or 1.15.\* cluster provisioned with the [required resources](#hardware-requirements)
+-   The [NGINX Ingress Controller](https://github.com/helm/charts/tree/master/stable/nginx-ingress) correctly set up in your cluster
 -   A [PostgreSQL server](#postgresql-server-setup) accessible from the Kubernetes cluster
 
-## Cluster resource requirements
+## Hardware requirements
 
-To have an enjoyable experience with Codacy, you should have the
-following calculations in mind when allocating resources for the
-installation and defining the number of concurrent analysis.
+The following high-level architecture is important in understanding how Codacy uses and allocates hardware resources.
 
-Without accounting for analysis ([next section](#analysis)),
-you should need at least the following resources:
+You can look at Codacy separately as the "Platform" and the "Analysis" cluster. The Platform contains the UI and other
```

Comment:
Maybe use bold for **Platform** and **Analysis** here to make them stand out and get reader attention